dracut-initramfs-restore.sh

```diff
@@ -46,8 +46,15 @@ elif mountpoint -q /efi; then
 elif mountpoint -q /boot/efi; then
     IMG="/boot/efi/$MACHINE_ID/$KERNEL_VERSION/initrd"
 else
-    echo "No initramfs image found to restore!"
-    exit 1
+    files=("/boot/initr*${KERNEL_VERSION}*")
```

Comment:
How is this guaranteed to work? Someone could set it to be `$KERNEL_VERSION-initramfs`...